fetch/client/src/main/scala/endpoints4s/fetch/EndpointsSettings.scala

```diff
@@ -3,38 +3,45 @@ package endpoints4s.fetch
 import endpoints4s.Hashing
 
 import scala.annotation.nowarn
+import scala.concurrent.duration.FiniteDuration
 
 /** Settings for XHR interpreter.
   * @param baseUri Base of the URI of the service that implements the endpoints, can be absolute or relative (e.g. "http://foo.com" or "/bar")
   */
-final class EndpointsSettings private (val baseUri: Option[String]) extends Serializable {
+final class EndpointsSettings private (val baseUri: Option[String], val timeout: Option[FiniteDuration]) extends Serializable {
 
   override def toString =
-    s"EndpointsSettings($baseUri)"
+    s"EndpointsSettings($baseUri, $timeout)"
 
   override def equals(other: Any): Boolean =
     other match {
       case that: EndpointsSettings =>
-        baseUri == that.baseUri
+        baseUri == that.baseUri && timeout == that.timeout
       case _ => false
     }
 
   override def hashCode(): Int = Hashing.hash(baseUri)
```

Comment:
You need to add the timeout here as well

Reply:
Good catch, I just added it